fili-core/src/main/java/com/yahoo/bard/webservice/web/handlers/workflow/DruidWorkflow.java

```diff
@@ -114,7 +119,8 @@ public DataRequestHandler buildWorkflow() {
         if (CacheFeatureFlag.TTL.isOn()) {
             handler = new CacheRequestHandler(handler, dataCache, mapper);
         } else if (CacheFeatureFlag.LOCAL_SIGNATURE.isOn()) {
-            handler = new CacheV2RequestHandler(handler, dataCache, querySigningService, mapper);
+            handler =
+                    new CacheV2RequestHandler(handler, dataCache, querySigningService, querySignedCacheService, mapper);
```

Comment:
Ideally the wrapping on this would have the 'new CacheV2...andler(' on the same line as handler= and then each of the remaining terms double indented ennding int ');' on it's own line aligned with 'handler ='